bears/haskell/HaskellLintBear.py

```diff
@@ -19,7 +21,9 @@ class HaskellLintBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='hlint')}
+    REQUIREMENTS = {CabalRequirement(
+        package='hlint', version='1.9.35-1'),
```

Comment:
`1.9.35-1` isnt useful. that is an apt version specification
`1.9.35` is the correct version specifier for `cabal install`